model/schema/bulkload.yml

```diff
@@ -0,0 +1,198 @@
+id: https://github.com/alliance-genome/agr_curation_schema/src/schema/bulkload.yaml
+
+imports:
+  - core
+
+classes:
+
+  BulkLoadGroup:
+    mixins:
+      - AuditedObject
```

Comment:
I realize that the AuditedObject class requires a 'created_by' and a 'modified_by' entry; will these always be available for BulkLoad, BulkLoadGroup, and BulkLoadFile objects? If yes, then OK, but if not then we either need to relax the requirements or something else.

Reply:
They will be set via whoever is logged into the UI.